.github/workflows/diff_shades.yml

```diff
@@ -24,7 +24,7 @@ jobs:
 
       - name: Install diff-shades and support dependencies
         run: |
-          python -m pip install click packaging urllib3
+          python -m pip install 'click<8.1.0' packaging urllib3
```

Comment:
Why do we pin < 8.1.0 if this fixes the bug? What am I missing - Is this for diff-shades?

Reply:
There's another bug, https://github.com/pallets/click/issues/2227

Reply:
Click 8.1.0 breaks mypyc (and probably mypy in general) as it both deleted internal APIs we import and also had a bad typing change (Jelle would have the link).

Reply:
Ahh ok - I just read this below and was super confused as to why we pinned: 
```markdown
- Fix Black to work with Click 8.1.0 (#2966)
```
Cheers